app/webpack/reactjs/components/open_studios/open_studios_info_form.tsx

```diff
@@ -30,6 +32,55 @@ function denullify(participant) {
   return result;
 }
 
+interface SpecialEventScheduleFieldsProps {
+  specialEvent: types.SpecialEventDetails;
+}
+
+interface TimeSlot {
+  start: string;
+  end: string;
+}
+interface TimeSlotCheckBoxProps {
+  timeslot: TimeSlot;
+  name: string;
+}
+
+const generateSlotFieldName = (timeslot: string): string =>
+  `videoConferenceSchedule[${timeslot}]`;
+
+const formatTimeSlot = (timeslot: types.TimeSlot): string => {
```

Comment:
note to self: Look into some clean up here.  These components and functions are kind of just dumped here.  Maybe they belong here or maybe somewhere else

Reply:
Pulled this out into it's own file